package.json

```diff
@@ -9,25 +9,50 @@
     "open data",
     "open knowledge"
   ],
-  "main": "index.js",
+  "main": "lib/index.js",
   "dependencies": {
-    "bluebird": "^2.9.30",
     "csv": "^0.4.2",
     "d3-time-format": "^0.3.2",
     "moment": "^2.13.0",
     "superagent": "^1.2.0",
```

Comment:
@zhenyab wondering if we can cleanly replace this and use native fetch API. In other libs we have tried to do this and use `isomorphic-fetch` to polyfill fetch on Node.

- https://github.com/openspending/os-packager/blob/master/package.json#L48
- https://developer.mozilla.org/en/docs/Web/API/Fetch_API

Here is an example of where I have used fetch, for reference: https://github.com/okfn/data-quality-dashboard/blob/master/app/utils.js#L28

WDYT?